internal/wasi/wasi.go

```diff
@@ -80,19 +80,39 @@ const (
 	ImportClockTimeGet = `(import "wasi_snapshot_preview1" "clock_time_get"
     (func $wasi.clock_time_get (param $id i32) (param $precision i64) (param $result.timestamp i32) (result (;errno;) i32)))`
 
-	FunctionFdAdvise             = "fd_advise"
-	FunctionFdAllocate           = "fd_allocate"
-	FunctionFdClose              = "fd_close"
-	FunctionFdDataSync           = "fd_datasync"
-	FunctionFdFdstatGet          = "fd_fdstat_get"
-	FunctionFdFdstatSetFlags     = "fd_fdstat_set_flags"
-	FunctionFdFdstatSetRights    = "fd_fdstat_set_rights"
-	FunctionFdFilestatGet        = "fd_filestat_get"
-	FunctionFdFilestatSetSize    = "fd_filestat_set_size"
-	FunctionFdFilestatSetTimes   = "fd_filestat_set_times"
-	FunctionFdPread              = "fd_pread"
-	FunctionFdPrestatGet         = "fd_prestat_get"
-	FunctionFdPrestatDirName     = "fd_prestat_dir_name"
+	FunctionFdAdvise   = "fd_advise"
+	FunctionFdAllocate = "fd_allocate"
+
+	// FunctionFdClose closes a file descriptor.
+	// See https://github.com/WebAssembly/WASI/blob/snapshot-01/phases/snapshot/docs.md#fd_close
+	FunctionFdClose = "fd_close"
+	// ImportFdClose is the WebAssembly 1.0 (MVP) Text format import of FunctionFdClose
+	ImportFdClose = `(import "wasi_snapshot_preview1" "fd_close"
+    (func $wasi.fd_close (param $fd i32) (result (;errno;) i32)))`
+
+	FunctionFdDataSync         = "fd_datasync"
+	FunctionFdFdstatGet        = "fd_fdstat_get"
+	FunctionFdFdstatSetFlags   = "fd_fdstat_set_flags"
+	FunctionFdFdstatSetRights  = "fd_fdstat_set_rights"
+	FunctionFdFilestatGet      = "fd_filestat_get"
+	FunctionFdFilestatSetSize  = "fd_filestat_set_size"
+	FunctionFdFilestatSetTimes = "fd_filestat_set_times"
+	FunctionFdPread            = "fd_pread"
+
+	// FunctionFdPrestatGet returns the prestat data of a file descriptor.
+	// See https://github.com/WebAssembly/WASI/blob/snapshot-01/phases/snapshot/docs.md#fd_prestat_get
+	FunctionFdPrestatGet = "fd_prestat_get"
+	// ImportFdPrestatGet is the WebAssembly 1.0 (MVP) Text format import of FunctionFdPrestatGet
+	ImportFdPrestatGet = `(import "wasi_snapshot_preview1" "fd_prestat_get"
+    (func $wasi.fd_prestat_get (param $fd i32) (param $result.prestat i32) (result (;errno;) i32)))`
+
+	// FunctionFdPrestatDirName returns the path of the prestat directory of a file descriptor.
```

Comment:
what is prestat? (I don't know why it is named in that way in the spec though :D)

```suggestion
	// FunctionFdPrestatDirName returns the path of the preopended directory of a file descriptor.
```

Reply:
nice catch! They were simply my typos...